17:22:17.182 INFO Test execution started: capture
17:22:17.182 INFO Browser: chrome
17:22:17.183 INFO Environment: test_env
17:22:17.183 INFO Using data:
    url: http://{}:{}@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer=
17:22:20.361 INFO Navigate to: 'http://User1:Automation_1_3@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer='
17:22:28.732 INFO Waiting for 3 seconds
17:22:47.108 ERROR ElementNotFound: Element //td[text()='20191001000219'] not found using selector xpath:'//td[text()='20191001000219']'
Traceback (most recent call last):
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\test_runner\test_runner.py", line 173, in run_test
    self.test_module.test(execution.data)
  File "C:\Users\alexandr.may\testdir\projects\some_project\tests\capture.py", line 20, in test
    print(a.get_element_text(new_order_page_form.runningMan('20191001000219')))
  File "C:\Users\alexandr.may\testdir\projects\some_project\pages\new_order_page_form.py", line 43, in runningMan
    return golemDriver.find(xpath="//td[text()='{}']".format(key))
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\extended_driver.py", line 177, in find
    return common._find(self, **kwargs)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\common.py", line 122, in _find
    element_name, timeout, wait_displayed)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\common.py", line 47, in _find_webelement
    .format(element_name, selector_type, selector_value))
golem.core.exceptions.ElementNotFound: Element //td[text()='20191001000219'] not found using selector xpath:'//td[text()='20191001000219']'

17:22:47.472 INFO Close browser
17:22:49.633 INFO Test Result: CODE ERROR
